src/test/setup_common.h

```diff
@@ -38,9 +38,21 @@ extern FastRandomContext g_insecure_rand_ctx;
  */
 extern bool g_mock_deterministic_tests;
 
-static inline void SeedInsecureRand(bool deterministic = false)
+enum class SeedRand {
+    ZEROS, //!< Seed with a compile time constant of zeros
+    SEEDR, //!< Call the Seed() helper
```

Comment:
Is this intentionally abbreviated as `SEEDR`, or should it be `SEED`/`SEEDER`?

Reply:
Happy to change it to something else if you can find a better name

Reply:
I was thinking simply `SEED`? :)

Reply:
Changed. Let's hope this doesn't conflict with any windows symbols